test/test_cli_integration.py

```diff
@@ -13,7 +13,7 @@
 
 # Not coverage tested since integration tests doesn't run in
 # the same build step
-class CliIntegrationTest(unittest.TestCase): # pragma: no cover
+class CliIntegrationTest(unittest.TestCase):  # pragma: no cover
```

Comment:
what is the extra space here?

Reply:
Is it problematic?
I actually did not notice it

Reply:
nope not